pages/index.js

```diff
@@ -16,29 +19,48 @@ export default function Home() {
 
       <Main>
         <h1>My App</h1>
-        <SearchbarLostItem />
+        <div>
+          <h4>Search1</h4>
+          <input
+            placeholder="What are you looking for?"
+            onChange={(event) => setSearch1(event.target.value)}
+          ></input>
+        </div>
         <ul>
-          {lostItems.map((lostItem) => (
-            <LostItem
-              name={lostItem.name}
-              description={lostItem.description}
-              img={lostItem.img}
-              key={lostItem.id}
-              id={lostItem.id}
-            />
-          ))}
+          {lostItems
+            .filter((lostItem) => lostItem.name.toLowerCase().includes(search1))
+            .map((lostItem) => (
+              <LostItem
+                name={lostItem.name}
+                description={lostItem.description}
+                img={lostItem.img}
+                key={lostItem.id}
+                id={lostItem.id}
+              />
+            ))}
         </ul>
         <h2>Second Array of Dummy Data begins here:</h2>
+        <div>
+          <h4>Search2</h4>
+          <input
+            placeholder="What are you looking for?"
+            onChange={(event) => setSearch2(event.target.value)}
+          ></input>
```

Comment:
you `input` needs a corresponding `label` for accessibility reasons. More [info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)